### PR TITLE
Implement issue https://github.com/vanadiumlabs/arbotix_ros/issues/24:

### PR DESCRIPTION
**Pull request by corot**
Allow 16 bit values on arbotix_msgs/Analog messages, but assume 8 bits by default. So changes should be transparent for current users except for the extra size of message value.

### CODE REVIEW

File: arbotix_python/bin/arbotix_driver [hunk 1]
```diff
@@ -128,7 +128,8 @@ class ArbotixROS(ArbotiX):
                     pin = rospy.get_param('~'+v+'/'+name+'/pin',1)
                     value = rospy.get_param('~'+v+'/'+name+'/value',0)
                     rate = rospy.get_param('~'+v+'/'+name+'/rate',10)
-                    self.io[name] = t(name, pin, value, rate, self)
+                    leng = rospy.get_param('~'+v+'/'+name+'/leng',1)  # just for analog sensors
```
Comment (mikeferguson) [inline arbotix_python/bin/arbotix_driver]: Parameter names are rarely shortened, so this should really be spelled out fully as "length". (the name on the server that is, not the python variable name)

**Comment by corot** [comment]
:+1:

**Comment by corot** [comment]
Any problem with the PR?  Should I change something else?

**Comment by mikeferguson** [comment]
Nope. I just forgot to come back and merge it.

**Pull request merged**